helm_deploy/create-and-vary-a-licence/values.yaml

```diff
@@ -10,7 +10,9 @@ cronjobs:
   expire_licences:
     enabled: true
     schedule: "0 4 * * *" # 4am every day
-
+  deactivate_release_date_passed_licences:
+    enabled: false
+    schedule: "30 1 * * 1-5" # 1:30am UTC Monday - Friday
```

Comment:
can we change this to 2 am, Monday to Friday? just so it doesn't run at the same time as the timeout job

Reply:
fixed